packages/logs/README.md

```diff
@@ -1,119 +1,388 @@
+| `clientToken`         | String  | Yes      |                 | A [Datadog client token][2].                                                                             |
+| `site`                | String  | Yes      | `datadoghq.com` | The Datadog site of your organization. US: `datadoghq.com`, EU: `datadoghq.eu`                           |
+| `service`             | String  | No       |                 | The service name for your application.                                                                   |
+| `env`                 | String  | No       |                 | The application’s environment, for example: prod, pre-prod, staging, etc.                                |
+| `version`             | String  | No       |                 | The application’s version, for example: 1.2.3, 6c44da20, 2020.02.13, etc.                                |
+| `forwardErrorsToLogs` | Boolean | No       | `true`          | Set to `false` to stop forwarding console.error logs, uncaught exceptions and network errors to Datadog. |
+| `sampleRate`          | Number  | No       | `100`           | The percentage of sessions to track: `100` for all, `0` for none. Only tracked sessions send rum events. |
+| `silentMultipleInit`  | Boolean | No       |                 | Prevent logging errors while having multiple init.                                                       |
+
+Options that must have a matching configuration when using the `RUM` SDK:
+
+| Parameter                      | Type    | Required | Default | Description                                                                                                                                                  |
+|--------------------------------|---------|----------|---------|--------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `trackSessionAcrossSubdomains` | Boolean | No       | `false` | Preserve the session across subdomains for the same site.                                                                                                    |
+| `useSecureSessionCookie`       | Boolean | No       | `false` | Use a secure session cookie. This disables RUM events sent on insecure (non-HTTPS) connections.                                                              |
+| `useCrossSiteSessionCookie`    | Boolean | No       | `false` | Use a secure cross-site session cookie. This allows the logs SDK to run when the site is loaded from another one (iframe). Implies `useSecureSessionCookie`. |
+
+## Usage
+
+### Custom logs
+
+When the Datadog browser log library is initialized, send a custom log entry directly to Datadog with the API:
+
+```
+logger.debug | info | warn | error (message: string, messageContext = Context)
+```
+
+{{< tabs >}}
+{{% tab "NPM" %}}
```

Comment:
Tabs markup is showing up in the README rendered by Github. Could we improve this?